homeassistant/components/knx/__init__.py

```diff
@@ -188,6 +206,14 @@ async def async_setup(hass, config):
         schema=SERVICE_KNX_SEND_SCHEMA,
     )
 
+    async_register_admin_service(
+        hass,
+        DOMAIN,
+        SERVICE_KNX_EVENT_REGISTER,
+        hass.data[DOMAIN].service_event_register_modify,
```

Comment:
It would be convenient to store the `KNXModule` instance in a local variable, besides in `hass.data` so we don't need to access `hass.data` many times in this function just to access that instance.

This can be cleaned up later.